js/src/config.js

```diff
@@ -20,6 +20,10 @@ export const LOG_KEYS = {
   TransferModalStore: {
     path: 'modals/Transfer/store',
     desc: 'Transfer Modal MobX Store'
+  },
+  Signer: {
+    path: 'secuerApi',
```

Comment:
`secureApi` - would also actually like consistency, i.e. path to actual location/entrypoint as per above entry.)